core-web/libs/sdk/react/src/lib/components/Container/Container.spec.tsx

```diff
@@ -31,6 +31,8 @@ describe('Container', () => {
         expect(screen.getByTestId('no-component')).toHaveTextContent(
             'No Component for content-type-1'
         );
+
+        expect(screen.getByTestId('empty-container')).toHaveTextContent('This container is empty.');
```

Comment:
Can we have both cases in the same container?

Reply:
No, i added other test for this case